pkg/medusa/reconcile.go

```diff
@@ -246,6 +246,10 @@ func medusaEnvVars(medusaSpec *api.MedusaClusterTemplate, dcConfig *cassandra.Da
 			Name:  "MEDUSA_MODE",
 			Value: mode,
 		},
+		{
+			Name:  "MEDUSA_TMP_DIR",
+			Value: "/var/lib/cassandra",
```

Comment:
Do we want to group the downloads into a Medusa sub-folder here? Or is Medusa going to put a sub-folder in this directory. My only concern with dumping directly into `/var/lib/cassandra` is dirtying up this directory. If Medusa will put restore downloads in sub-folders under this directory, that eases my concerns.